velox/common/base/StatsReporter.h

```diff
@@ -135,6 +135,9 @@ class BaseStatsReporter {
   virtual void addHistogramMetricValue(folly::StringPiece key, size_t value)
       const = 0;
 
+  /// Return the metrics in a serialized string format.
```

Comment:
/// Return the aggregated metrics in a serialized string format.

Reply:
I'm wondering if this API should be const. Depending on implementations, clean-like operations can happen inside per each fetch interval.

Reply:
Makes sense to remove the const. I address the comments. Thanks!